csrankings.csv

```diff
@@ -9024,6 +9030,7 @@ Joel P. Ilao,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty
 Joel Reardon,University of Calgary,http://pages.cpsc.ucalgary.ca/~joel.reardon,NOSCHOLARPAGE
 Joel Spencer,New York University,http://cs.nyu.edu/spencer,AmmWYgEAAAAJ
 Joel Voldman,Massachusetts Institute of Technology,http://www.rle.mit.edu/biomicro,NOSCHOLARPAGE
+Joelle Pineau,McGill University,https://www.cs.mcgill.ca/~jpineau,CEt6_mMAAAAJ

```

Comment:
Appointment must be >= 75% over the academic year to qualify for inclusion.

Reply:
Not sure what it means, but Ok I will let know Joelle Pineau.